src/screens/SendReceive/Send/AssetsScreen/AssetRow.tsx

```diff
@@ -91,23 +90,16 @@ const AssetRow = ({ asset, style, isLast }: AssetRowProps) => {
     setIsSelected(isNowSelected)
 
     if (isNowSelected) {
-      inputRef.current?.focus()
+      setTimeout(() => inputRef.current?.focus(), 500)
```

Comment:
I spent a lot time on this... This is the hack I found so that the focus happens after the animation finishes. I suspect that it's due to the animation that the keyboard was not showing. This "fix" seems to work.